pygem/nurbshandler.py

```diff
@@ -205,20 +222,386 @@ def write(self, mesh_points, filename, tolerance=None):
 			wire = wire_maker.Wire()
 
 			# trimming the surfaces
-			brep_surf = BRepBuilderAPI_MakeFace(occ_face.GetHandle(), wire).Shape()
+			brep_surf = BRepBuilderAPI_MakeFace(occ_face.GetHandle(),
+												wire).Shape()
 			compound_builder.Add(compound, brep_surf)
 			n_faces += 1
 			faces_explorer.Next()
 		self.write_shape_to_file(compound, self.outfile)
 
+
+	def parse_face(self, topo_face):
```

Comment:
You never use self in this method. Make it a static method (see quantified code). Also the methods below have the same problem.